mailpoet/lib/API/JSON/v1/Tags.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\API\JSON\v1;
+
+use MailPoet\API\JSON\Endpoint as APIEndpoint;
+use MailPoet\Entities\TagEntity;
+use MailPoet\InvalidStateException;
+use MailPoet\Tags\TagRepository;
+
+class Tags extends APIEndpoint {
+
+  private $repository;
+
+  public function __construct(
+    TagRepository $repository
+  ) {
+    $this->repository = $repository;
+  }
+
+  public function save($data = []) {
```

Comment:
I think this method may better be called `create`. See [newsletters](https://github.com/mailpoet/mailpoet/blob/78446d174a228b8be34d651469df9b96d447357f/mailpoet/lib/API/JSON/v1/Newsletters.php#L385), for instance, how `save` vs. `create` is used. That said, I'm not sure are always consistent.

Reply:
446569cd9